### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by erickma1**
Add a contact form to the mobile design
Add styling to the contact form in style.css
@Aqurds and @Peter-Kibet added additional styling to the stylesheet style.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -264,6 +265,39 @@ <h2>Skills</h2>
...
+            <article class="contact-article">
+                <p>
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </p>
+                <form action="https://formspree.io/f/xyyaopza" method="POST" class="contact-form">
+                    <input name="FullName" type="text" placeholder="Full name" required maxlength="30" />
+                    <input name="Email" type="email" placeholder="Email address" required />
+                    <textarea name="Message" rows="4" cols="50" placeholder="Write me something..." maxlength="500" required></textarea>
+                    <div class="contact-button">
+                        <button type="submit">Get in touch</button>
```
Comment (bobb-Rob) [inline index.html:278]: - [ ] Please implement the submit button color change on mouse hover. 

image

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @erickma1, 

Good job so far! 👏👏👏 . I must admit that you dis this project excellently well. But there are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by erickma1** [comment]
Implement the submit button color change on mouse hover.

**Review by braincee — APPROVED** [review]
Hi Team @erickma1, @Peter-Kibet and @Aqurds 👋,  I am @braincee, your Code Reviewer.

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

youre-approved-shirley-auto-group

OPTIONAL CHANGES
---

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @braincee._

---

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by usorfaitheloho — COMMENTED** [review]
Hi @erickma1  👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**